manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml

```diff
@@ -52,3 +52,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  - serviceaccounts/finalizers
+  - secrets
```

Comment:
1. We don't need get/list privileges for secrets. We only need to know the *_name_* of the secret for the purposes of volume mounting, which is already in the service account object:
```
$ k get sa foobar -o yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  creationTimestamp: "2019-08-20T18:26:32Z"
  name: foobar
  namespace: default
  resourceVersion: "395208"
  selfLink: /api/v1/namespaces/default/serviceaccounts/foobar
  uid: 091f2460-c378-11e9-97ea-025000000001
secrets:
- name: foobar-token-kv2mt <<<<<<<<<< here
```

Reply:
I realized we still need secret for [the workflow persistence](https://github.com/argoproj/argo/blob/b9b87b7fa0cd3177c2b89cacff189f4893c5af95/workflow/persist/sqldb/sqldb.go#L72)...
I think we should add the role separately in the manifests.